<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceControlBuilder</class>
 <widget class="QMainWindow" name="FaceControlBuilder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>346</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>541</width>
    <height>346</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>541</width>
    <height>346</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Face Control Builder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>44</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="C_characterDisplayLabel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>{$Character}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>218</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>13</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>8</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Temp Locator Version:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>13</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>8</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Temp Locator Path:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="22,78">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <widget class="QComboBox" name="C_TempLocatorComboBox">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="C_TempLocatorLineEdit">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>24</height>
               </size>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="70,15,15">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="C_importLoacatorButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>23</height>
               </size>
              </property>
              <property name="font">
               <font>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>import</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="C_publishLocatorButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>23</height>
               </size>
              </property>
              <property name="font">
               <font>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Publish</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>13</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Temp Locators</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="C_createLocatorsButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Create  Temp  Locators</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>3</number>
            </property>
            <item>
             <widget class="QPushButton" name="C_mirrorLocatorButtonLR">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>L  -&gt;  R</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="C_mirrorLocatorButtonRL">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>R  -&gt;  L</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>13</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Mirror Temp Locators</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="C_buildControlButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>26</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Build   Control</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>3</number>
            </property>
            <item>
             <widget class="QPushButton" name="C_importDataButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>26</height>
               </size>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Import   Control   Data</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="C_publishDataButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>26</height>
               </size>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Export   Control   Data</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionVersionChanged">
   <property name="text">
    <string>versionChanged</string>
   </property>
  </action>
  <action name="actionImportTempLocators">
   <property name="text">
    <string>importTempLocators</string>
   </property>
  </action>
  <action name="actionPublishTempLocators">
   <property name="text">
    <string>publishTempLocators</string>
   </property>
  </action>
  <action name="actionCreateTempLocators">
   <property name="text">
    <string>createTempLocators</string>
   </property>
  </action>
  <action name="actionMirrorLocatorsLR">
   <property name="text">
    <string>mirrorLocatorsLR</string>
   </property>
  </action>
  <action name="actionMirrorLocatorsRL">
   <property name="text">
    <string>mirrorLocatorsRL</string>
   </property>
  </action>
  <action name="actionBuildFaceControl">
   <property name="text">
    <string>buildFaceControl</string>
   </property>
  </action>
  <action name="actionCurrentCharacterChanged">
   <property name="text">
    <string>currentCharacterChanged</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>C_TempLocatorComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>actionVersionChanged</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>C_importLoacatorButton</sender>
   <signal>clicked()</signal>
   <receiver>actionImportTempLocators</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>C_publishLocatorButton</sender>
   <signal>clicked()</signal>
   <receiver>actionPublishTempLocators</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>C_createLocatorsButton</sender>
   <signal>clicked()</signal>
   <receiver>actionCreateTempLocators</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>C_mirrorLocatorButtonLR</sender>
   <signal>clicked()</signal>
   <receiver>actionMirrorLocatorsLR</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>C_mirrorLocatorButtonRL</sender>
   <signal>clicked()</signal>
   <receiver>actionMirrorLocatorsRL</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>C_buildControlButton</sender>
   <signal>clicked()</signal>
   <receiver>actionBuildFaceControl</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
